microservices.yml

```diff
@@ -451,37 +461,127 @@ commands:
             # run dangerfile
             ./node_modules/.bin/danger ci --failOnErrors
 
-  publish-unit-test-results-to-xray:
+  publish-unit-test-results-via-xray:
     description: Publish all unit test results to Jira Xray
     steps:
+      - attach_workspace:
+          at: .
       - run:
-          name: Download publish to Xray script
+          name: Upload Mocha unit test results to Jira Xray
           when: always
           command: |
-            # get push-test-results-to-xray.sh
-             wget https://raw.githubusercontent.com/MGMDV-Orbs/microservices/f/xray-r2.3/.circleci/push-test-results-to-xray.sh -O push-test-results-to-xray.sh
-      - run:
-          name: Upload Mocha unit test results to Jira Xray
+            TEST_RESULTS_FILE=./reports/unit-test-mocha-results.xml
+            PROJECT_CODE=SRV
+            TEST_TYPE=junit
```

Comment:
Seems like this variable is only used in one place and that is to echo to stdout for informational purposes.
does not seem like it is used for any flow control or re-use

Probably a good idea to not initialize unnecessary variables